docs/concepts/oltp.md

```diff
@@ -5,89 +5,75 @@ could be payments, sales, car sharing rides, game scores, or API usage.
+One of the most common ways to scale systems is to horizontally scale or shard them. This means
 different servers process different sets of transactions. Unfortunately, business transactions don't
-shard well.
+shard well. Horizontal scaling is a poor fit for OLTP:
 
-Horizontal scaling works poorly for OLTP because most accounts cannot be neatly partitioned between
-shards. Transactions between accounts on different shards become even more complex and slow. Row
-locks on hot accounts are even worse when the transactions must execute across shards.
+- Most accounts cannot be neatly partitioned between shards.
+- Transactions between accounts on different shards become more complex and slow.
+- Row locks on hot accounts worsen when the transactions must execute across shards.
 
-Another approach to scaling OLTP systems is to use
-[MapReduce](https://en.wikipedia.org/wiki/MapReduce) for billing. Unfortunately, this makes it
-almost impossible to provide real-time balance reporting or real-time spending limits, because
-balances are only calculated periodically in batches. This is a poor user experience that cannot
-readily be fixed after the system is designed.
+Another approach to scaling OLTP systems is to use MapReduce for billing. But this makes it hard to
+provide real-time balance reporting or spending limits. It also creates a poor user experience
+that's hard to fix post system design.
 
-TigerBeetle intentionally uses a single-core design, along with numerous unique performance
-optimizations, to provide high throughput without the downsides of horizontal scaling.
+TigerBeetle uses a [single-core design](./performance.md#single-threaded-by-design )
+and unique performance optimizations to deliver high throughput. And this without the downsides of
+horizontal scaling.
 
 ## Bottleneck for Your System
 
-You can only do as much business as your database supports. As such, you need a core OLTP database
-capable of handling your transactions today, on your busiest days, and for the decades to come.
-
-TigerBeetle is designed to handle **1 million transactions per second**. For comparison, the entire
-Visa network processes an average of around 9,000 transactions per second[^1] and supports a maximum
-of around 65,000 transactions per second[^2].
-
-[^1]:
-    The
-    [Visa Fact Sheet](https://usa.visa.com/dam/VCOM/global/about-visa/documents/aboutvisafactsheet.pdf)
-    reports that the network processed 276.3 billion transactions in 2023.
+You can only do as much business as your database supports. You need a core OLTP database capable of
+handling your transactions on your busiest days. And for decades to come.
 
-[^2]:
-    Visa.
-    [_A deep dive on Solana, a high performance blockchain network_](https://usa.visa.com/solutions/crypto/deep-dive-on-solana.html). 2023.
+TigerBeetle is designed to handle **1 million transactions per second**, to remove the risk of your
+business outgrowing your database.
```

Comment:
Removed this whole TPS contest, for two reasons: 

- We ourselves don't yet have a clear benchmark which we can point to, as a measure of TPS. 
- The public numbers we use feel a bit dodge (Solana?), and they are going to change quickly, but the concepts should stand for eternity!

Instead, I worked-in the `risk` idea, as I think it is actually a rather key motivation for performance.